.gitignore

```diff
@@ -1,6 +1,7 @@
 # Misc
 #-----
 .DS_Store
+!/addOns/spider/src/test/resources/**/.DS_Store
```

Comment:
Do I need to chmod these read only to prevent Macs from clobbering or updating them?